linera-views/src/batch.rs

```diff
@@ -121,6 +121,9 @@ impl UnorderedBatch {
         &mut self,
         db: &DB,
     ) -> Result<(), DB::Error> {
+        if self.key_prefix_deletions.is_empty() {
+            return Ok(());
+        }
```

Comment:
(This seems unnecessary, since the loop below will just be empty. Doesn't hurt either, of course.)